### PR TITLE
Add SWR organization resource

### LINKED ISSUE #428: Any plan to support SWR (Software Repository for Container) ?
On the Terraform doc, I couldn't find a resource provider for provisioning SWR resource... Unless I missed it on the doc, is there any plan to support SWR ;D? 
Thanks heaps.

### PR DESCRIPTION
**Pull request by niuzhenguo**
**What this PR does / why we need it**:

Added new resource SWR organization.

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #428 

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud' TESTARGS='-run=TestAccSWROrganization_basic'
TF_ACC=1 go test ./huaweicloud/ -v -run=TestAccSWROrganization_basic -timeout 360m -parallel 4
=== RUN   TestAccSWROrganization_basic
=== PAUSE TestAccSWROrganization_basic
=== CONT  TestAccSWROrganization_basic
--- PASS: TestAccSWROrganization_basic (77.64s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud       77.660s
```

### CODE REVIEW

File: huaweicloud/endpoints.go [hunk 1]
```diff
@@ -100,6 +100,11 @@ var allServiceCatalog = map[string]ServiceCatalog{
 		Name:    "functiongraph",
 		Version: "v2",
 	},
+	"swrv2": {
```
Comment (ShiChangkuo) [inline huaweicloud/endpoints.go]: suggest to use `swr`, then we can export it in `provider/endpoints`

File: huaweicloud/endpoints.go [hunk 2]
```diff
@@ -100,6 +100,11 @@ var allServiceCatalog = map[string]ServiceCatalog{
...
+		Name:             "swr-api",
+		Version:          "v2",
+		WithOutProjectID: true,
+	},
```
Comment (ShiChangkuo) [inline huaweicloud/endpoints.go:107]: should add a acc test in endpoints_test.go

File: huaweicloud/resource_huaweicloud_swr_organization.go [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"creator": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"permission": {
```
Comment (ShiChangkuo) [inline huaweicloud/resource_huaweicloud_swr_organization.go:43]: shall we add `registry` attribute which users can push images
the format of registry is  `swr.{region}.myhuaweicloud.com/{name}`

File: huaweicloud/endpoints.go (hunk 1 above)
Comment (niuzhenguo) [reply huaweicloud/endpoints.go]: done

File: huaweicloud/endpoints.go (hunk 2 above)
Comment (niuzhenguo) [reply huaweicloud/endpoints.go:107]: done

File: huaweicloud/resource_huaweicloud_swr_organization.go (hunk 3 above)
Comment (niuzhenguo) [reply huaweicloud/resource_huaweicloud_swr_organization.go:43]: added a login_server url instead as what Azure does.

File: huaweicloud/resource_huaweicloud_swr_organization.go [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+	case 1:
+		permission = "Read"
+	}
+
+	d.Set("region", GetRegion(d, config))
+	d.Set("name", n.Name)
+	d.Set("creator", n.CreatorName)
+	d.Set("permission", permission)
+
+	login := fmt.Sprintf("swr.%s.myhuaweicloud.com", GetRegion(d, config))
```
Comment (ShiChangkuo) [inline huaweicloud/resource_huaweicloud_swr_organization.go]: a tinny tips: please use c.Cloud to replace `myhuaweicloud.com` as the same way in endpoints_test

Comment (niuzhenguo) [reply huaweicloud/resource_huaweicloud_swr_organization.go]: done

**Pull request merged**